src/main/asciidoc/inc/image/_naming.adoc

```diff
@@ -50,6 +50,9 @@ When specifying the container name pattern the following placeholders can be use
 | *%i*
 | An index which is incremented if a container has already been created. With this parameter it is easily possible to have multiple, similar containers. See the example below for more details.
 
+| *%r*
```

Comment:
Is it required that %r is given as a single argument ? What happens to any additional character added to the pattern ? Are they just ignored ? If so, then  %r is not the same like the other "placeholders" as it just indicates a different behaviour that e.g. no name should be set at all. While the feature to have random containers names is desirable, its probably not a good fit to introduce it via a placeholder pattern.